src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -25,6 +31,8 @@ const DaysLeft = () => {
         '',
       );
 
+      setFirstDay(firstDay);
+      setLastDay(lastDay);
```

Comment:
```suggestion
      setFirstDay(first);
      setLastDay(last);
```

mirror change from above